src/app/parcels/form/ParcelForm.tsx

```diff
@@ -142,6 +146,12 @@ const ParcelForm: React.FC<ParcelFormProps> = ({
         }
     }, [clientDetails, clientIdForFetch]);
 
+    useEffect(() => {
+        if (editMode && !packingSlotShown) {
+            initialFormErrors.packingSlot = Errors.invalidPackingSlot;
+            setFormErrors(initialFormErrors);
```

Comment:
I don't think we actually want to modify the `initialFormErrors` prop (idk if it even modifies, given React behaviour).

Instead i think we should use the current state of errors, and modify just the packing slot.
```suggestion
            setFormErrors(currentState => { 
                ...currentState, 
                packingSlot = Errors.invalidPackingSlot
            });
```